cdk/README.md

```diff
@@ -1,5 +1,5 @@
 ## Infrastructure
-Install Node.js 16 using `nvm` (Node Version Manager):
+Install Node.js 18 using `nvm` (Node Version Manager):
```

Comment:
Are we changing the local Node version the CDK uses to deploy infrastructure too, not just the application Node version? This is probably fine but should do the same thing in igvfd so the versions are consistent.

Reply:
Node 16 becomes unsupported on [September 11](https://endoflife.date/nodejs), so I think it makes sense to switch everything over now.